### PR TITLE
OP-406 increase coverage in dicomtype; format Manager file

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Coverage before:
image

Coverage after:
image

### CODE REVIEW

**Review by mwithi — APPROVED** [review]
LGTM.
We passed 1000+ tests! 🚀 Thanks @dbmalkovsky !

```
Results :

Tests run: 1010, Failures: 0, Errors: 0, Skipped: 1
```

File: src/test/java/org/isf/dicomtype/test/Tests.java [hunk 1]
```diff
@@ -62,6 +74,143 @@ public void testDicomTypeSets() throws Exception {
 		_checkDicomTypeIntoDb(code);
 	}
 
+	@Test
+	public void testIoGetDicomType() throws Exception {
+		String typeId = _setupTestDicomType(false);
+		DicomType dicomType = dicomTypeIoOperationRepository.findOne(typeId);
```
Comment (emecas) [inline src/test/java/org/isf/dicomtype/test/Tests.java:80]: this line is not used!!!

File: src/test/java/org/isf/dicomtype/test/Tests.java [hunk 2]
```diff
@@ -62,6 +74,143 @@ public void testDicomTypeSets() throws Exception {
...
+	public void testIoIsCodePresent() throws Exception {
+		DicomType dicomType = new DicomType("id", "description");
+		assertThat(dicomTypeIoOperation.newDicomType(dicomType)).isTrue();
+		assertThat(dicomTypeIoOperation.isCodePresent(dicomType.getDicomTypeID())).isTrue();
+	}
+
+	@Test
+	public void testMgrGetDicomType() throws Exception {
+		String typeId = _setupTestDicomType(false);
+		DicomType dicomType = dicomTypeIoOperationRepository.findOne(typeId);
```
Comment (emecas) [inline src/test/java/org/isf/dicomtype/test/Tests.java:121]: this line is not used!!!

**Review by emecas — APPROVED** [review]
Thanks @dbmalkovsky 

LGTM

```
Results :

Tests run: 766, Failures: 0, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:02 min
[INFO] Finished at: 2021-01-21T21:27:04-05:00
[INFO] ------------------------------------------------------------------------
```

File: src/test/java/org/isf/dicomtype/test/Tests.java (hunk 1 above)
Comment (mwithi) [reply src/test/java/org/isf/dicomtype/test/Tests.java:80]: @dbmalkovsky +1

File: src/test/java/org/isf/dicomtype/test/Tests.java (hunk 2 above)
Comment (mwithi) [reply src/test/java/org/isf/dicomtype/test/Tests.java:121]: @dbmalkovsky +1

File: src/test/java/org/isf/dicomtype/test/Tests.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/test/java/org/isf/dicomtype/test/Tests.java]: Fixed.

File: src/test/java/org/isf/dicomtype/test/Tests.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/test/java/org/isf/dicomtype/test/Tests.java]: Fixed.

File: src/test/java/org/isf/dicomtype/test/Tests.java (hunk 1 above)
Comment (mwithi) [reply src/test/java/org/isf/dicomtype/test/Tests.java]: Thanks a lot!

File: src/test/java/org/isf/dicomtype/test/Tests.java (hunk 2 above)
Comment (mwithi) [reply src/test/java/org/isf/dicomtype/test/Tests.java]: Thanks a lot!

**Pull request merged**